war/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  parser: "postcss-less",
+  plugins: [
+    require('autoprefixer')
```

Comment:
So I think by default autoprefixer uses the browser list from package.json (or other files), and then if not defined, does its own list of browsers.

I think we need to make sure we have a browserlist (https://www.npmjs.com/package/browserslist) defined that matches the browser compatibility matrix that matrix jenkins - https://www.jenkins.io/doc/administration/requirements/web-browsers/ (since its a relative list, its pretty easy to define in browserlist)

Reply:
There's one on package.json https://github.com/jenkinsci/jenkins/blob/f5fdc4bc4c0e11cc47f21475cd137d8eda2ba3e2/war/package.json#L54

Reply:
oh cool, i totally missed that when i scaned

👍 👍